docs/_docs/props.md

```diff
@@ -167,7 +167,18 @@ MyComponent.create(c)
    .name("Three")
 ```
 
-Note that, when using `varArg`, `resType`s are not supported.
+If you specific `resType`:
+
+```java
+MyComponent.create(c)
+   .namePx(1f)
```

Comment:
`name` is probably not a great example here as it's a string resource. Could you add a spec with a resource type to illustrate this as above?

Reply:
It's done I replace it with `textSize`